seckey.h

```diff
@@ -80,16 +80,16 @@ class VariableRounds
 	//! \throws InvalidRounds if the number of rounds are invalid
 	inline void ThrowIfInvalidRounds(int rounds, const Algorithm *alg)
 	{
-#if defined(__BORLANDC__)
-		if (rounds < MIN_ROUNDS || rounds > MAX_ROUNDS)
-			throw InvalidRounds(alg ? alg->AlgorithmName() : std::string("VariableRounds"), rounds);
-#elif (M==INT_MAX)	// Coverity and result_independent_of_operands
-		if (rounds < MIN_ROUNDS)
-			throw InvalidRounds(alg ? alg->AlgorithmName() : "VariableRounds", rounds);	
-#else
-		if (rounds < MIN_ROUNDS || rounds > MAX_ROUNDS)
-			throw InvalidRounds(alg ? alg->AlgorithmName() : "VariableRounds", rounds);
-#endif
+		if (M == INT_MAX) // Coverity and result_independent_of_operands
+		{
+			if (rounds < MIN_ROUNDS)
+				throw InvalidRounds(alg ? alg->AlgorithmName() : std::string("VariableRounds"), rounds);
+		}
+		else
+		{
+			if (rounds < MIN_ROUNDS || rounds > MAX_ROUNDS)
+				throw InvalidRounds(alg ? alg->AlgorithmName() : std::string("VariableRounds"), rounds);
+		}
 	}
 

```

Comment:
> This was _most likely_ because of the `M == INT_MAX` preprocessor

Yeah, that's what I am thinking too.

I'd prefer to ensure both MSVC and Borland are clean. If I had to select one and sacrifice the other then MSVC trumps Borland. Hopefully Crayon has the time to perform the test for us.